### PR TITLE
maintainers/team-list: add cloudposse team for atmos

### PR DESCRIPTION
**Pull request by nitrocode**
## Description of changes

- To allow for additional maintainers for the atmos package, we can use the official cloudposse team to keep it maintained.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [x] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [x] `sandbox = relaxed`
  - [x] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [x] (Package updates) Added a release notes entry if the change is major or breaking
  - [x] (Module updates) Added a release notes entry if the change is significant
  - [x] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by nitrocode** [comment]
cc: @dudymas @osterman

File: pkgs/applications/networking/cluster/atmos/default.nix [hunk 1]
```diff
@@ -37,6 +37,6 @@ buildGoModule rec {
     changelog = "https://github.com/cloudposse/atmos/releases/tag/v${version}";
     description = "Universal Tool for DevOps and Cloud Automation (works with terraform, helm, helmfile, etc)";
     license = licenses.asl20;
-    maintainers = with maintainers; [ rb ];
```
Comment (dotlambda) [inline pkgs/applications/networking/cluster/atmos/default.nix:40]: Why is @nitrocode being removed?

Comment (nitrocode) [reply pkgs/applications/networking/cluster/atmos/default.nix:40]: Yes!

**Comment by nitrocode** [comment]
cc: @dotlambda please review once more

**Comment by nitrocode** [comment]
cc: @dotlambda friendly ping

@osterman @dudymas please weigh in if this looks appropriate from slack discussions

**Comment by dotlambda** [comment]
Why do we need a team if it's only used in one package?

**Comment by dotlambda** [comment]
Please squash the last two commits and change the order of the then remaining ones.

**Comment by dudymas** [comment]
This looks good to me. I mentioned in slack that @mcalhoun might want to also be added to the team.

**Comment by nitrocode** [comment]
cc: @dotlambda can we merge this first and then the rest of the cloudposse team can add additional users when needed?

**Comment by nitrocode** [comment]
cc: @dotlambda @LeSuisse @roberth

**Review by roberth — APPROVED** [review]

**Comment by roberth** [comment]
Thank you for your commitment to help out with package maintenance! :heart:

**Pull request merged**